<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QMainWindow" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>705</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>731</width>
    <height>705</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>705</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scorbot ATS</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection Settings</string>
    </property>
    <widget class="QLineEdit" name="ConnectionStatus">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>21</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Disconnect">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Connect">
     <property name="geometry">
      <rect>
       <x>108</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ScanPorts">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>21</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Scan ports...</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="LoadProgram">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Program</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RunProgram">
     <property name="geometry">
      <rect>
       <x>539</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Program</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ReadProgram">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>76</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Read Program</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>711</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Program Editor</string>
    </property>
    <widget class="QPlainTextEdit" name="ProgramEditorBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearScreenProgramEditor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Screen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OpenProgram">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SaveProgram">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>711</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Console</string>
    </property>
    <widget class="QPlainTextEdit" name="ConsoleBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>691</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearScreenConsole">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Screen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendConsole">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>170</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
